wrap/CMakeLists.txt

```diff
@@ -26,34 +26,15 @@ install(FILES cmake/gtwrapConfig.cmake cmake/PybindWrap.cmake
               cmake/GtwrapUtils.cmake
         DESTINATION "${SCRIPT_INSTALL_DIR}/gtwrap")
 
+include(GNUInstallDirs)
+# Install the gtwrap python package as a directory so it can be found for
+# wrapping.
+install(DIRECTORY gtwrap DESTINATION "${CMAKE_INSTALL_DATADIR}/gtwrap")
+
 # Install wrapping scripts as binaries to `CMAKE_INSTALL_PREFIX/bin` so they can
 # be invoked for wrapping.
-# We use DESTINATION (instead of TYPE) so we can support older CMake versions.
-install(PROGRAMS scripts/pybind_wrap.py scripts/matlab_wrap.py
-        DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(PROGRAMS scripts/pybind_wrap.py scripts/matlab_wrap.py TYPE BIN)
```

Comment:
Should we be using TYPE? I left a comment saying that we need DESTINATION for older versions of CMake.

Reply:
Agreed :) I'll push a change to this to wrap (but let's first merge this)